app/controllers/base_controller.rb

```diff
@@ -3,9 +3,10 @@ class BaseController < ApplicationController
 
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
-  after_action :verify_authorized, except: :index
+  after_action :verify_authorized
 
```

Comment:
We should default to verifying that scopes are called for index actions, and other actions call authorize (or explicitly skip) so using Policies is enforced
eg
```
after_action :verify_authorized, :except => :index
after_action :verify_policy_scoped, :only => :index
```